<% if resource.errors.any? %>
  <div id="error_explanation">
    <div class="alert alert-danger">
      下記の<%= resource.errors.count %>つのエラーがあります。
    </div>
    <ul>
    <% resource.errors.full_messages.each do |msg| %>
      <li class="text-danger"><%= msg %>。</li>
    <% end %>
    </ul>
  </div>
<% end %>
